outputs.tf

```diff
@@ -13,7 +13,12 @@ resource "local_file" "jenkins_infra_data_report" {
       "outbound_ips" = concat(split(",", module.infra_ci_outbound_sponsorship.public_ip_list), split(",", module.privatek8s_outbound.public_ip_list)),
     },
     "privatek8s.jenkins.io" = {
-      "outbound_ips" = split(",", module.privatek8s_outbound.public_ip_list),
+      "outbound_ips"      = split(",", module.privatek8s_outbound.public_ip_list),
+      "private_lb_subnet" = "private-vnet-data-tier",
```

Comment:
shouldn't it be `privatek8s-tier` ?

Reply:
Nope: https://github.com/jenkins-infra/kubernetes-management/blob/bdabecdcc2787afa2fe07f2e320323b095be8913/config/private-nginx-ingress_privatek8s.yaml#L5 specifies the name of the subnet in which to create the private LB.

Reply:
so this one is wrong line 22 ? `"private_lb_subnet" = "privatek8s-sponsorship-tier",`

Reply:
It's not wrong either: it's the name of the subnet, in the new "sponsorship" vnet in which we seek to create the private LB for the new cluster: https://github.com/jenkins-infra/azure-net/blob/324174236247809f03cad72d76c97fbc0b588f95/vnets.tf#L186

For comparison, the current `privatek8s` clusters has its private LB in the following subnet: https://github.com/jenkins-infra/azure-net/blob/324174236247809f03cad72d76c97fbc0b588f95/vnets.tf#L106

Reply:
the size difference between those 2 subnets make me wonder

Reply:
The first one is used by more things than only the `privatek8s` cluster: it's also used by VPN, and other things (hence the bigger size).

The second one was dimensioned by you and is only aimed at the new cluster.

Reply:
pure luck 🤪